michelleegg-seattle.md

```diff
@@ -0,0 +1,5 @@
+# Recommendations for Seattle, WA
+
+### Places to eat

```

Comment:
I've added a pair off the top of my head, hope you'll enjoy them when you're in the area.